include/torch-mlir/Conversion/Utils/Utils.h

```diff
@@ -100,6 +100,17 @@ void computeBroadcastShape(ConversionPatternRewriter &rewriter, Location loc,
                            SmallVector<int64_t> &resultShape,
                            SmallVector<Value> &resultShapeValue);
 
+// Helper function for expand index tentors, the following expandSizes().
+SmallVector<int64_t> inferSizesFromTwoTensor(PatternRewriter &rewriter,
+                                             Location loc,
+                                             SmallVector<int64_t> a,
+                                             SmallVector<int64_t> b);
+
+// expand index tentors shape into same size for indices concatnation.
```

Comment:
Could you please extend the explanation, maybe with an example?

Reply:
done